src/main/java/com/fasterxml/jackson/core/JsonParser.java

```diff
@@ -138,6 +138,20 @@ public enum Feature {
          */
         ALLOW_SINGLE_QUOTES(false),
 
+        /**
+         * Feature that determines whether parser will allow use 
+         * of the RS control character (exhibited by 0x1E and parsed as a
+         * tab or /t) within parsed content or not. 
+         *<p>
+         * Since JSON specification requires use of double quotes for
```

Comment:
copy paste?

Reply:
sorry, comments were an afterthought. Changed comments to more accurately reflect the feature